docs/Setup_Kubernetes_Cluster.md

```diff
@@ -185,215 +389,92 @@ daemonset.apps/kube-flannel-ds created
 > kubectl delete -f https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml
 ```
 
-- [WeaveNet](https://github.com/weaveworks/weave)
+### [WeaveNet](https://github.com/weaveworks/weave) **(E.O.L)**
```

Comment:
i guess i should replace all examples needs to be updated with cilium instead. weave works okay, but not recommended.